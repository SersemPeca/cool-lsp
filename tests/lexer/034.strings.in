"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
"same string"
