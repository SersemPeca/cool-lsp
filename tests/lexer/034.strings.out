#1 STR_CONST "same string"
#2 STR_CONST "same string"
#3 STR_CONST "same string"
#4 STR_CONST "same string"
#5 STR_CONST "same string"
#6 STR_CONST "same string"
#7 STR_CONST "same string"
#8 STR_CONST "same string"
#9 STR_CONST "same string"
#10 STR_CONST "same string"
#11 STR_CONST "same string"
#12 STR_CONST "same string"
#13 STR_CONST "same string"
#14 STR_CONST "same string"
#15 STR_CONST "same string"
#16 STR_CONST "same string"
#17 STR_CONST "same string"
#18 STR_CONST "same string"
#19 STR_CONST "same string"
#20 STR_CONST "same string"
#21 STR_CONST "same string"
#22 STR_CONST "same string"
#23 STR_CONST "same string"
#24 STR_CONST "same string"
